sys-apps/portage/portage-3.0.20-r6.ebuild

```diff
@@ -1,28 +1,31 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-# Flatcar: Based on portage-3.0.8.ebuild from commit
-# be7749525a3958edbcecee314d0e1c294222f87f in Gentoo repo (see
-# https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-apps/portage/portage-3.0.8.ebuild?id=be7749525a3958edbcecee314d0e1c294222f87f).
+# Flatcar: Based on portage-3.0.20-r6.ebuild from commit
+# 71967da8a7c26c9b6744af4ea7ee68dee0d31795 in Gentoo repo (see
+# https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-apps/portage/portage-3.0.20-r6.ebuild?id=71967da8a7c26c9b6744af4ea7ee68dee0d31795).
 
 EAPI=7
 
-DISTUTILS_USE_SETUPTOOLS=no
+DISTUTILS_USE_SETUPTOOLS=bdepend
 PYTHON_COMPAT=( pypy3 python3_{6..7} )
```

Comment:
Yeah, I hope this will work. I remember I picked a version of portage that supported python 3.6. Portage 3.0.20 probably supports 3.7 or higher. I planned to update portage after I'm done with updating our python situation (dropping python2, packaging recent python3 version and maybe moving some packages back to portage-stable).